qutebrowser/browser/network/networkmanager.py

```diff
@@ -241,12 +241,21 @@ def clear_rejected_ssl_errors(self, url):
     @pyqtSlot('QNetworkReply', 'QAuthenticator')
     def on_authentication_required(self, reply, authenticator):
         """Called when a website needs authentication."""
-        answer = self._ask("Username ({}):".format(authenticator.realm()),
+        import netrc
+        net = netrc.netrc()
+        authenticators = net.authenticators(reply.url().host())
+        user, password = None, None
+
+        if authenticators:
+            (user, account, password) = authenticators
```

Comment:
Please also use `_account` as it's unused, and add a `# pylint: disable=unpacking-non-sequence` to hide a pylint false-positive.